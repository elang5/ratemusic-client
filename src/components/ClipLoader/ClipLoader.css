.loader-container {
  display: flex;
  justify-content: center;
}

.loader-container {
  margin-top: 2rem;
}