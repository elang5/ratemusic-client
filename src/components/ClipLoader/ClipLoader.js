import React from "react";
import { ClipLoader } from "react-spinners";
import "./ClipLoader.css";

export default function Loader(props) {
  return (
    <div className="loader-container">
      <ClipLoader
        className="loader"
        sizeUnit={"px"}
        size={150}
        color={"rgba(252, 61, 57, 0.73)"}
        loading={props.loading}
      />
    </div>
  );
}
