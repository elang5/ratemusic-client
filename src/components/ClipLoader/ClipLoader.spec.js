import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import ClipLoader from './ClipLoader'

describe(`ClipLoader Component`, () => {

  it(`renders the ClipLoader component`, () => {
    const wrapper = shallow(<ClipLoader />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})