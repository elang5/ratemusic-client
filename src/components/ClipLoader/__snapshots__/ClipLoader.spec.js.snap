// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ClipLoader Component renders the ClipLoader component 1`] = `
<div
  className="loader-container"
>
  <onlyUpdateForKeys(Loader)
    className="loader"
    color="rgba(252, 61, 57, 0.73)"
    css={Object {}}
    loading={true}
    size={150}
    sizeUnit="px"
  />
</div>
`;
